setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup, find_packages
+import sys
+import os
+import os.path as op
+from glob import glob
+
+template_files = glob(op.join('jupyter_book', 'book_template', '**', '*'), recursive=True)
```

Comment:
```suggestion
template_files = glob(op.join('book_template', '**', '*'), recursive=True)
```

Following the shablona example, I think that this might need to be a path relative to the package.

Reply:
Alternatively, if that doesn't work:

```suggestion
template_files = [op.join('book_template', '*'), op.join('book_template', '*', '*')]
```

And so on, for as many levels of recursion as you need.